src/cpu/aarch64/brgemm/brgemm_containers.hpp

```diff
@@ -95,6 +95,28 @@ struct brgemm_kernel_container_t {
     std::map<const brgemm_t *, const brgemm_kernel_t *> brgemm_map_;
 };
 
+struct brgemm_palette_container_t {
```

Comment:
Palettes are for Intel AMX. You probably may not want to copy-paste them.

Reply:
@dzarukin Thank you for your feedback. I've made the suggested modifications. Please let me know if there are any additional changes needed.

Reply:
Hi @abhijain1204fujitsu, thank you for reminding and sorry for a delay, I'll promote it soon.